File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        if(nums.length==0){
            return 0;
        }
        int unique=0;
        for(int i= unique+1;i<nums.length;i++){
            if(nums[i]!=nums[unique]){
                nums[unique+1]=nums[i];
                unique++;
            }
        }
        return unique+1;
    }
}

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int index=0;
        if(nums.size()==0){
            return 0;
        }
        for(int i = index+1;i<nums.size();i++){
            if(nums[i]!=nums[index]){
                nums[index+1]=nums[i];
                index++;
            }
        }
        return index+1;
    }
};